crd/multitenancy/api/v1alpha1/multitenantpodnetworkconfig.go

```diff
@@ -61,6 +61,21 @@ type MultitenantPodNetworkConfigStatus struct {
 	MacAddress string `json:"macAddress,omitempty"`
 	// Gateway IP
 	GatewayIP string `json:"gatewayIP,omitempty"`
+	// InterfaceDetails
+	InterfaceInfos []InterfaceInfo `json:"interfaceInfo,omitempty"`
+}
+
+type InterfaceInfo struct {
```

Comment:
(non-blocking naming suggestion) This might be better named `NetworkInterface`. `Info` is superfluous in the name, since all structs carry information. `Interface` alone might be confusing because of the Go keyword `interface`.

Reply:
It's already committed in another PR .